frontend/app/routes/public/renew/$id/adult-child/update-home-address.tsx

```diff
@@ -490,12 +490,29 @@ function AddressSuggestionDialogContent({ enteredAddress, suggestedAddress }: Ad
       />
       <DialogFooter>
         <DialogClose asChild>
-          <Button id="dialog.corrected-address-close-button" startIcon={faChevronLeft} disabled={fetcher.isSubmitting} variant="alternative" size="sm">
+          <Button
+            id="dialog.corrected-address-close-button"
+            startIcon={faChevronLeft}
+            disabled={fetcher.isSubmitting}
+            variant="alternative"
+            size="sm"
+            data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult_Child:Dialog Cancel - Home address click"
```

Comment:
The button text actually says "Back":

```suggestion
            data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult_Child:Dialog Back - Home address click"
```

Reply:
Fixed text